sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/siprouting/implementation/SipRoutingAdminClientImpl.java

```diff
@@ -0,0 +1,390 @@
+    }
+
+    /** The HTTP pipeline to send requests through. */
+    private final HttpPipeline httpPipeline;
+
+    /**
+     * Gets The HTTP pipeline to send requests through.
+     *
+     * @return the httpPipeline value.
+     */
+    public HttpPipeline getHttpPipeline() {
+        return this.httpPipeline;
+    }
+
+    /** The serializer to serialize an object into a string. */
+    private final SerializerAdapter serializerAdapter;
+
+    /**
+     * Gets The serializer to serialize an object into a string.
+     *
+     * @return the serializerAdapter value.
+     */
+    public SerializerAdapter getSerializerAdapter() {
+        return this.serializerAdapter;
+    }
+
+    /**
+     * Initializes an instance of SipRoutingAdminClient client.
+     *
+     * @param endpoint The communication resource, for example https://resourcename.communication.azure.com.
+     * @param apiVersion Api Version.
+     */
+    SipRoutingAdminClientImpl(String endpoint, String apiVersion) {
+        this(
+                new HttpPipelineBuilder()
+                        .policies(new UserAgentPolicy(), new RetryPolicy(), new CookiePolicy())
+                        .build(),
+                JacksonAdapter.createDefaultSerializerAdapter(),
+                endpoint,
+                apiVersion);
+    }
+
+    /**
+     * Initializes an instance of SipRoutingAdminClient client.
+     *
+     * @param httpPipeline The HTTP pipeline to send requests through.
+     * @param endpoint The communication resource, for example https://resourcename.communication.azure.com.
+     * @param apiVersion Api Version.
+     */
+    SipRoutingAdminClientImpl(HttpPipeline httpPipeline, String endpoint, String apiVersion) {
+        this(httpPipeline, JacksonAdapter.createDefaultSerializerAdapter(), endpoint, apiVersion);
+    }
+
+    /**
+     * Initializes an instance of SipRoutingAdminClient client.
+     *
+     * @param httpPipeline The HTTP pipeline to send requests through.
+     * @param serializerAdapter The serializer to serialize an object into a string.
+     * @param endpoint The communication resource, for example https://resourcename.communication.azure.com.
+     * @param apiVersion Api Version.
+     */
+    SipRoutingAdminClientImpl(
+            HttpPipeline httpPipeline, SerializerAdapter serializerAdapter, String endpoint, String apiVersion) {
+        this.httpPipeline = httpPipeline;
+        this.serializerAdapter = serializerAdapter;
+        this.endpoint = endpoint;
+        this.apiVersion = apiVersion;
+        this.service =
+                RestProxy.create(SipRoutingAdminClientService.class, this.httpPipeline, this.getSerializerAdapter());
+    }
+
+    /**
+     * The interface defining all the services for SipRoutingAdminClient to be used by the proxy service to perform REST
+     * calls.
+     */
+    @Host("{endpoint}")
+    @ServiceInterface(name = "SipRoutingAdminClien")
+    private interface SipRoutingAdminClientService {
```

Comment:
Please add this configuration to your Swagger README configutation

`service-interface-as-public: true`

The configuration will configure Autorest to generate this interface as `public` instead of `private` which resolves issues with `SecurityManager`s.

Reply:
Added. https://github.com/Azure/azure-sdk-for-java/pull/26819/commits/4fb57b58377fa8640cfa2d0204b1c7457bfb1499